src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -152,34 +167,42 @@ def add_profile_args(parser):
     )
 
 
-def add_endpoint_args(parser):
+def _add_endpoint_args(parser):
     endpoint_group = parser.add_argument_group("Endpoint")
 
+    endpoint_group.add_argument(
```

Comment:
nit for later: PA core looks like it has `--endpoint` for specifying the HTTP route for openai like `/v1/chat/completions`

and this tool uses `--endpoint` as the same as `--protocol`. We should probably align on names where it makes sense later.